config/samples/glance_v1beta1_glanceapi.yaml

```diff
@@ -19,3 +19,11 @@ spec:
   replicas: 1
   storageRequest: 10G
   secret: glance-secret
+  #tls:
```

Comment:
Do we want to provide an example or a dedicated section in the `README` ? I think this is going to be a common pattern for all the operators, but having a bit of doc regarding these new parameters would help people understand how the feature works.

Reply:
I have started this PR for an addition on internal certs https://github.com/stuggi/docs/pull/new/tls_internal . should we still have something dedicated in the `README` ? just a link to the global conf. What kind of example you have in mind? a glance CR ?

Reply:
ah ok, I think having a general doc in that repo is better than having yet another dedicate section here where we explain the fields that are part of the new API definition.
Considering we're going to drive this process from the openstack-operator, I think we're good here. I'll take care about any extra doc in a follow up.

Reply:
nice, thanks